lib/blackbeard/feature.rb

```diff
@@ -4,12 +4,22 @@ class Feature < Storable
     include Chartable
 
     set_master_key :features
-    string_attributes :name, :description, :status
+    string_attributes :name, :description, :status, :threshold_array_serialized
     integer_attributes :visitors_rate, :users_rate
     json_attributes :group_segments
 
     has_many :metrics => 'Metric'
 
+    def threshold_array=(thresholds)
+      self.threshold_array_serialized = thresholds.join(",")
+    end
+
+    def threshold_array
```

Comment:
The getter should call the setter so that the first time, it is called the array is generated and saved.